modules/phase_field/examples/rigidbodymotion/AC_CH_advection_constforce_rect.i

```diff
@@ -0,0 +1,218 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 50
+  ny = 25
+  nz = 0
+  xmax = 50
+  ymax = 25
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[Variables]
+  [./c]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./w]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./eta]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[AuxVariables]
+  [./vadv00]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./vadv01]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./vadv0_div]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[Kernels]
+  [./c_res]
+    type = SplitCHParsed
+    variable = c
+    f_name = F
+    kappa_name = kappa_c
+    w = w
+    args = eta
+  [../]
+  [./w_res]
+    type = SplitCHWRes
+    variable = w
+    mob_name = M
+  [../]
+  [./time]
+    type = CoupledTimeDerivative
+    variable = w
+    v = c
+  [../]
+  # advection kernel corrsponding to CH equation
```

Comment:
Can you put those comments inside the blocks? Otherwise Peacock will scramble them up.